app/helpers/index.js

```diff
@@ -0,0 +1,36 @@
+const bcrypt = require('bcryptjs');
+const { encryptionError } = require('../errors');
+const saltRounds = 10;
+exports.hashPassword = obj =>
+  bcrypt
+    .hash(obj.password, saltRounds)
+    .then(password => ({ ...obj, password }))
+    .catch(error => {
+      throw encryptionError(error.message);
+    });
+
+// eslint-disable-next-line max-len
+const emailRegex = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/gim;
+
+exports.validations = toValidate => ({
```

Comment:
I see two concepts (validations and crypt) merged into one file, maybe one for each one is clearer